client/my-sites/sidebar/sidebar.jsx

```diff
@@ -345,19 +345,16 @@ export class MySitesSidebar extends Component {
 		const { canUserManageOptions, isJetpack, site, siteSuffix, translate } = this.props;
 		const storeLink = '/store' + siteSuffix;
 		const showStoreLink = config.isEnabled( 'woocommerce/extension-dashboard' ) &&
-			site && isJetpack && canUserManageOptions;
+			site && isJetpack && canUserManageOptions && this.props.isSiteAutomatedTransfer;
```

Comment:
`isSiteAutomatedTransfer` is not destructured above because `isSiteAutomatedTransfer` is already defined as a selector and the linter will gripe.